### PR TITLE
Release version `0.6.0` of SpatialIndex.

### PR DESCRIPTION
**Pull request by 1uc**

### CODE REVIEW

File: bluebrain/deployment/environments/applications_libraries.yaml [hunk 1]
```diff
@@ -25,6 +25,7 @@ spack:
         - py-scikit-learn
         - py-scipy
         - py-tensorflow
+        - py-tqdm
```
Comment (matz-e) [inline bluebrain/deployment/environments/applications_libraries.yaml]:
```suggestion
```

File: bluebrain/deployment/environments/applications_libraries.yaml [hunk 2]
```diff
@@ -65,6 +66,7 @@ spack:
     - py-scikit-learn
     - py-scipy
     - py-tensorflow
+    - py-tqdm
```
Comment (matz-e) [inline bluebrain/deployment/environments/applications_libraries.yaml]:
```suggestion
```

**Review by matz-e — COMMENTED** [review]
```
depends_on('py-tqdm', type=('build', 'run'), when='…')
```
is needed for a proper _runtime_ dependency. The default is linking dependency

File: bluebrain/deployment/environments/applications_libraries.yaml (hunk 1 above)
Comment (1uc) [reply bluebrain/deployment/environments/applications_libraries.yaml]: reverted.

File: bluebrain/deployment/environments/applications_libraries.yaml (hunk 2 above)
Comment (1uc) [reply bluebrain/deployment/environments/applications_libraries.yaml]: reverted.

File: bluebrain/repo-bluebrain/packages/spatial-index/package.py [hunk 3]
```diff
@@ -41,11 +40,11 @@ class SpatialIndex(PythonPackage):
     depends_on("py-mvdtool~mpi", type=("build", "run"))
     depends_on("py-numpy-quaternion", type=("build", "run"), when="@0.2.1:")
     depends_on("py-numpy", type=("build", "run"))
-    depends_on("py-tqdm", when="@0.5.3:")
+    depends_on("py-tqdm", type=("build", "run"), when="@0.5.3:")
 
     depends_on("boost@1.79.0:", when="@0.5.2:")
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/spatial-index/package.py]: stupid remark, but it would seem nicer if the Boost stuff is right next to each other

Comment (1uc) [reply bluebrain/repo-bluebrain/packages/spatial-index/package.py]: Yes, definitely.

**Review by matz-e — APPROVED** [review]

**Pull request merged**